burger.css

```diff
@@ -0,0 +1,50 @@
+   .header__nav {
+    width: 100%;
+    height: 100%;
+    position: fixed;
+    top: 0;
+    left: 0;
+    background: #E5E5E5;
+    z-index: 10;
+    padding: 186px;
+    transform: translateX(100%);
+   }
+.header__item {
+  width: 100%;
+  margin-right: 0;
+  margin-bottom: 84px;
+}
+
+.header__link {
+  font-size: 30px;
+  line-height: 36px;
+  color: black;
+}
+
+.header__nav-clouse {
+  width: 40px;
+  height: 40px;
+  position: absolute;
+  top: 20px;
+  right: 20px;
+  z-index: 11;
+}
+
+.header__nav-clouse-line {
+  display: block;
+  height: 3px;
+  background: black;
+  position: absolute;
+  top: 50%;
+  width: 100%;
+  
+}
+
+.header__nav-clouse-line:first-child {
+  transform: translateY(-50%) rotate(45eg);
+  }
+
+  .header__nav-clouse-line:last-child {
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/MyBali/pullRequest?prid=5781003)